app/utils/nonsipp/check/LandOrPropertyCheckStatusUtils.scala

```diff
@@ -43,11 +43,18 @@ object LandOrPropertyCheckStatusUtils {
       case _ =>
         journeysStartedList
           .map(
+//            index => {
+//              refineV[OneTo5000](index.toInt + 1).fold(
+//                _ => List.empty,
+//                refinedIndex => checkLandOrPropertyRecord(userAnswers, srn, refinedIndex)
+//              )
+//            }
+
             index => {
-              refineV[OneTo5000](index.toInt + 1).fold(
-                _ => List.empty,
-                refinedIndex => checkLandOrPropertyRecord(userAnswers, srn, refinedIndex)
-              )
+              refineV[OneTo5000](index.toInt + 1) match {
+                case Right(refinedIndex) => checkLandOrPropertyRecord(userAnswers, srn, refinedIndex)
```

Comment:
Are you sure this is the same logic?

Reply:
see comment on BondsCheckStatusUtils.scala